### PR TITLE
adds more units, orders in ui

### PR DESCRIPTION
**Pull request by tsop14**
- creates more units
- creates display name for units
- creates subsections to order selection on UI

- fixes styling on videos in original recipe body

screen shot 2016-10-13 at 15 58 53

### CODE REVIEW

**Comment by tsop14** [comment]
#49

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala [hunk 1]
```diff
@@ -85,27 +85,47 @@ case class TimesInMins(
 
 sealed trait CookingUnit {
   def abbreviation: String
+  def displayName: String
 }
 
 object CookingUnit {
   val unitMap = Map(
+    "cup" -> Cup,
```
Comment (LATaylor-guardian) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: The key in this map corresponds to the abbreviation on the corresponding object, correct? If so, we should reference Cup.abbreviation as the key here so we don't have it in two separate places.

**Comment by alastairjardine** [comment]
`FlOz` to `Fl oz`

Other than that 👍

**Comment by alastairjardine** [comment]
Oh Oh!

We need to add dessert spoon!

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala (hunk 1 above)
Comment (tsop14) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: ⚡️ V. sensible! Changed

**Comment by tsop14** [comment]
@alastairjardine dessert spoon added.

**Comment by LATaylor-guardian** [comment]
Fine from a code point of view 👍

**Pull request merged**